browser/src/control/Control.AutoFillPreviewTooltip.ts

```diff
@@ -0,0 +1,102 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+/*
+ * Control.AutoFillPreviewTooltip - class for tooltip of cell previews during auto fill
+ */
+
+/* global app */
+
+class AutoFillPreviewTooltip extends L.Control.AutoCompletePopup {
+	usageText: string;
+	newPopupData: PopupData;
+
+	constructor(map: ReturnType<typeof L.map>) {
+		super('autoFillPreviewTooltip', map);
+		this.newPopupData = {
```

Comment:
This JSON is almost the same as parent one.
Can we just modify different fields from here?

```
this.newPopupData,canHaveFocus = false;
this.newPopupData,noOverlay = true;
this.newPopupData,isAutoFillPreviewTooltip = true;
```

Reply:
removed `newPopupData` and added only different fields into `onAdd()`.

Done. Thanks!